sse2neon.h

```diff
@@ -5692,6 +5692,23 @@ FORCE_INLINE double _mm_cvtsd_f64(__m128d a)
 #endif
 }
 
+// Convert the lower double-precision (64-bit) floating-point element in a to a
+// 64-bit integer, and store the result in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtsd_si64
+FORCE_INLINE int64_t _mm_cvtsd_si64(__m128d a)
+{
+#if defined(__aarch64__)
+    return vgetq_lane_s64(vreinterpretq_s64_m128d(a), 0);
+#else
+    return ((int64_t *) &a)[0];
+#endif
```

Comment:
The implementation is wrong.
It just treats the data store in `double` as `int64_t`.
The data itself is not changed.

What you expect is like `3.0` -> `3`.
However, the `3.0` data stored in double would not be `0x03`.